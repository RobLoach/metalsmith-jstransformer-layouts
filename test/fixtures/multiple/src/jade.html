---
layout: _jadelayout.jade
pretty: true
---
This is my site using Jade!
