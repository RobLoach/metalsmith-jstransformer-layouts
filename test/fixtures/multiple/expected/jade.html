<!DOCTYPE html>
<html>
  <body>This is my site using Jade!
</body>
</html>